core/src/main/java/com/linecorp/armeria/common/util/SettableIntSupplier.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.util;
+
+import java.util.function.IntSupplier;
+
+import com.linecorp.armeria.client.limit.ConcurrencyLimit;
+import com.linecorp.armeria.common.annotation.UnstableApi;
+
+/**
+ * Supplies the cached value that is useful below cases.
```

Comment:
nit; not a complete sentence

```suggestion
 * Supplies a cached value which can be useful for the below case.
```